site/pages/smart-contracts/modular-account-v2/managing-ownership.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Managing ownership
+description: Managing ownership on your Modular Account V2
+---
+
+# Managing Ownership
+
+You can add an owner to your account, or transfer ownership of your account with Modular Account V2. Adding an owner has the same flow as adding a session key with no root permissions.
```

Comment:
very small grammar nit, but i would either remove the first comma or add another comma after "of your account"